### ISSUE TITLE
[Typo] Typo in SVD-hyperdrive notebook

### ISSUE THREAD

**Issue by loomlike** [microsoft/recommenders#838]
### Description
This is very tiny issue ;-)
At Section 3,
> 1. Download data and split into training and testing sets

but the data are split into three sets, training / validation / testing.

Also would be good to describe where will the validation set be used -- to validate hyperparameter selections.
